apps/src/lib/node/ledger/events.rs

```diff
@@ -13,23 +15,23 @@ use crate::facade::tendermint_proto::abci::EventAttribute;
 
 /// Indicates if an event is emitted do to
 /// an individual Tx or the nature of a finalized block
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Eq, PartialEq)]
 pub enum EventLevel {
     Block,
     Tx,
 }
 
 /// Custom events that can be queried from Tendermint
 /// using a websocket client
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Eq, PartialEq)]
```

Comment:
we can update the docstring (`from Tendermint using a websocket client` no more 😄 )